src/commands/deploy.rs

```diff
@@ -147,7 +147,8 @@ impl DeployCommand {
             self.hippo_username.clone(),
             self.hippo_password.clone(),
         )
-        .await?
+        .await
+        .context("Problem login Hippo")?
```

Comment:
A couple of things here:

* First, and apologies for nit-picking the English here, but this should be phrased as something like "Problem logging into Hippo"
* Second, it would be nice to extract the reason from the Hippo failure response and present that.  What this does is address the second part of #614 (say that the error comes from Hippo rather than from Bindle), which is good, but the reason is still buried in the raw JSON.  What I'd suggest is that we parse the JSON, look at either the `title` or `detail` elements (@bacongobbler can probably recommend which) and output that as part of the message.  We can still dump the JSON, but this way we get a top-level message which is still precise but is also more readable.

Hope this makes sense, happy to discuss further!